
class RepoNotFoundException(Exception):
    pass


class PullNotMergeableException(Exception):
    pass
